README.md

```diff
@@ -61,4 +61,4 @@ In general we want to preserve useful comments in the project configuration file
 Please revert unwanted deletions before committing them.
 
 #### Windows Driver
-See https://docs.espressif.com/projects/esp-idf/en/latest/esp32s3/get-started/establish-serial-connection.html#connect-esp32-s3-to-pc for driver installation guide.
+See [`https://docs.espressif.com`]https://docs.espressif.com/projects/esp-idf/en/latest/esp32s3/get-started/establish-serial-connection.html#connect-esp32-s3-to-pc for driver installation guide.
```

Comment:
I guess, that [using the USB peripheral](https://docs.espressif.com/projects/esp-idf/en/latest/esp32s3/get-started/establish-serial-connection.html#supported-usb-peripheral) (instead of the USB-to-UART bridge) does not require to install a driver.